config/complete/orderly-web.yml

```diff
@@ -78,6 +78,11 @@ orderly:
     ORDERLY_DB_PASS: VAULT:secret/db/password:value
 
 ## Api and Website configuration
+user-cli:
+  image:
+    repo: vimc
+    name: orderly-web-user-cli
+    tag: mrc-248
```

Comment:
how are these all being pushed to docker hub at the moment?

Reply:
https://github.com/vimc/orderly-web/pull/55/files#diff-d1f85965cc5d5a8c69759fc0a1461eedR18